### PR TITLE
Auth48 fixes for QPACK

### PR DESCRIPTION
**Pull request by MikeBishop**
Addresses most of the open RFC Editor questions.

### CODE REVIEW

File: rfc9204.md [hunk 1]
```diff
@@ -523,9 +522,10 @@ it specifies any other value, or omits SETTINGS_QPACK_MAX_TABLE_CAPACITY from
 SETTINGS, the encoder must treat this as a connection error of type
 QPACK_DECODER_STREAM_ERROR.
 
-For HTTP/3 servers and HTTP/3 clients when 0-RTT is not attempted or is
-rejected, the maximum table capacity is 0 until the encoder processes a SETTINGS
-frame with a non-zero value of SETTINGS_QPACK_MAX_TABLE_CAPACITY.
+For clients not using 0-RTT data (whether 0-RTT is not attempted or is rejected)
+and for all HTTP/3 servers, the maximum table capacity is 0 until the encoder
+processes a SETTINGS frame with a non-zero value of
+SETTINGS_QPACK_MAX_TABLE_CAPACITY.
```
Comment (MikeBishop) [inline rfc9204.md:528]: By parallel with the previous paragraph's first sentence.

File: rfc9204.md [hunk 2]
```diff
@@ -1200,12 +1200,13 @@ This is possible even over the Transport Layer Security Protocol
...
-  capabilities, potentially only forcing an increased number of guesses to learn
-  the length associated with a given guess. Padding schemes also work directly
-  against compression by increasing the number of bits that are transmitted.
+<aside><t>
+Note: Padding schemes only provide limited protection against an attacker with
+these capabilities, potentially only forcing an increased number of guesses to
+learn the length associated with a given guess. Padding schemes also work
+directly against compression by increasing the number of bits that are
+transmitted.
+</t></aside>
```
Comment (MikeBishop) [inline rfc9204.md:1209]: I couldn't find a kramdown way to do this, so I just inlined the XML.

Comment (martinthomson) [reply rfc9204.md:1209]: Carsten mentioned that you can use `{:aside}` in another thread, but I don't know exactly how to do that, maybe...

```suggestion
Note: Padding schemes only provide limited protection against an attacker with
these capabilities, potentially only forcing an increased number of guesses to
learn the length associated with a given guess. Padding schemes also work
directly against compression by increasing the number of bits that are
transmitted.
{:aside}
```

Comment (cabo) [reply rfc9204.md:1209]: :aside and :quote are meant to go with blockquotes:

{:aside}
> Note: Padding schemes only provide limited protection against an attacker with
these capabilities, potentially only forcing an increased number of guesses to
learn the length associated with a given guess. Padding schemes also work
directly against compression by increasing the number of bits that are
transmitted.

Comment (MikeBishop) [reply rfc9204.md:1209]: Ah, I see.  Thank you!
```suggestion
{:aside}
> Note: Padding schemes only provide limited protection against an attacker with
these capabilities, potentially only forcing an increased number of guesses to
learn the length associated with a given guess. Padding schemes also work
directly against compression by increasing the number of bits that are
transmitted.
```

**Pull request merged**